lang/kotlin/kotlin.py

```diff
@@ -8,19 +8,19 @@
 
 # Java Modifies
 ctx.lists["user.code_keyword"] = {
-    "var": "var",
-    "val": "val",
-    "lateinit": "lateinit",
-    "public": "public",
-    "private": "private",
-    "protected": "protected",
-    "static": "static",
-    "synchronized": "synchronized",
-    "volatile": "volatile",
-    "transient": "transient",
-    "abstract": "abstract",
-    "interface": "interface",
-    "final": "final",
+    "var ": "var",
+    "val ": "val",
+    "lateinit ": "lateinit",
+    "public ": "public",
+    "private ": "private",
+    "protected ": "protected",
+    "static ": "static",
+    "synchronized ": "synchronized",
+    "volatile ": "volatile",
+    "transient ": "transient",
+    "abstract ": "abstract",
+    "interface ": "interface",
+    "final ": "final",
```

Comment:
These are on the wrong side: the keys are the spoken phrase, the values will be what's inserted.

Reply:
Thanks for pointing that out. Should be fixed now.